ubuntu/Dockerfile

```diff
@@ -6,14 +6,10 @@ LABEL maintainer="Hive Solutions <development@hive.pt>"
 ARG DEBIAN_FRONTEND=noninteractive

```

Comment:
let's instead make use of the latest ubuntu version that allowed python2 (change FROM instead) otherwise we will have many problems downstream